File: dayfour/afterbreak/python/TaskThree.py
average = 0
length = 0
sum = 0
for count in range(1,11):
	
	score = int(input("enter score:"))
	sum = sum + score
	length = length + 1
average = sum / length	
print("the average of scores is: ",average)
print("the sum of scores is: ",sum)

File: dayfour/afterbreak/javascript/TaskThree.js
const scanner = require("prompt-sync")();
	let average = 0;
	let length = 0;
	let sum = 0;
	for(let count = 1 ; count <= 10; count++){
	
	const Score = Number(scanner("enter score"));
	
	sum =sum + Score;
	length = length + 1;
	}
	average = sum /length;
	console.log("the average of ten scores is: "+ average);
	console.log("the sum of ten scores is: "+ sum);
